src/Selenium2Driver.php

```diff
@@ -921,11 +971,20 @@ public function wait(int $timeout, string $condition)
 
     public function resizeWindow(int $width, int $height, ?string $name = null)
     {
-        $window = $this->getWebDriverSession()->window($name ?: 'current');
-        \assert($window instanceof Window);
-        $window->postSize(
-            array('width' => $width, 'height' => $height)
-        );
+        if ($this->isW3C()) {
+            // This method doesn't exist in older versions of php-webdriver.
+            /** @phpstan-ignore-next-line variable.undefined */
+            $this->wdSession->window($name ? $name : 'current')->postRect(
```

Comment:
this must use `getWebDriverSession()` to have the proper error handling if the method is called on a non-started driver instead of crashing by calling a method on `null`